conf/test.config

```diff
@@ -10,20 +10,23 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    resourceLimits = [
+        cpus: 4,
+        memory: '15.GB',
+        time: '1.h'
+    ]
+}
+
 params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function'
-
-    // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
-    max_time   = '6.h'
-
-    // Input data
-    input          = "$projectDir/assets/samplesheets/test_input_channels.yml"
-    mash_screen_db = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/genome/genome.fasta'
-
-    outdir     = 'results'
-    tracedir   = "${params.outdir}/pipeline_info"
-    publish_dir_mode = 'symlink'
+    input = "$projectDir/assets/samplesheets/test_samplesheet.csv"
```

Comment:
That might be fine for now, but later that should be rather in nf-core testdata repo